relevanceai/_api/endpoints/api_client.py

```diff
@@ -112,3 +112,58 @@ def _are_fields_in_schema(self, fields, dataset_id, schema=None):
                 f"{', '.join(invalid_fields)} are invalid fields. They are not in the dataset schema."
             )
         return
+
+    def _is_field_vectorized(self, field):
+        return
+
+    #validate section
+    def _validate_vector_field(self, field, include_chunk=False, guess_field=False):            
+        if field.endswith(field, "_vector_"):
+            return True
+        elif include_chunk and field.endswith(field, "_chunkvector_"):
+            return True
+        else:
+            return False
+
+    def _validate_metric(self, metric, automate=False):
```

Comment:
For clarity - can you add `automate: bool=False` because I don't know if it's meant to be a boolean or not otherwise until I read the logic - would help me add to it